src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingStatementTests.cs

```diff
@@ -2533,21 +2533,15 @@ void B()
 }
 ";
             CreateCompilation(source).VerifyDiagnostics(
-                // (6,8): error CS1003: Syntax error, '(' expected
+                // (6,8): error CS1031: Type expected
```

Comment:
>// (6,8): error CS1031: Type expected [](start = 16, length = 37)

The diagnostic quality here isn't the best. You could also type a parenthesis. Maybe add a PROTOTYPE marker to revisit.